addons/sale_crm/models/sale_order.py

```diff
@@ -13,3 +13,23 @@ class SaleOrder(models.Model):
 
     def action_confirm(self):
         return super(SaleOrder, self.with_context({k:v for k,v in self._context.items() if k != 'default_tag_ids'})).action_confirm()
+
+    def _message_post_after_hook(self, new_message, message_values):
```

Comment:
let us keep the same signature as in sale_order:
`def _message_post_after_hook(self, message, msg_vals)`